internal/webhooks/clusterclass.go

```diff
@@ -99,6 +103,21 @@ func (webhook *ClusterClass) ValidateUpdate(_ context.Context, oldObj, newObj ru
 
 // ValidateDelete implements validation for ClusterClass delete.
 func (webhook *ClusterClass) ValidateDelete(ctx context.Context, obj runtime.Object) error {
+	clusterClass, ok := obj.(*clusterv1.ClusterClass)
```

Comment:
Can you add `Fixes https://github.com/kubernetes-sigs/cluster-api/issues/5498` to the PR description?